tests/UI/specs/UIIntegration_spec.js

```diff
@@ -550,9 +550,20 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
             expect(await page.screenshotSelector('.pageWrap,.dataTable')).to.matchImage('ecommerce_overview');
         });
 
-        it('should load the ecommerce log page', async function () {
+        it.only('should load the ecommerce log page', async function () {
```

Comment:
```suggestion
        it('should load the ecommerce log page', async function () {
```

Probably not a change intended to be kept.